packages/x-components/src/components/global-x-bus.vue

```diff
@@ -34,11 +34,12 @@
       const subscription = reduce(
         this.$listeners,
         (subscription, eventName, callback) => {
-          return subscription.add(
+          subscription.add(
```

Comment:
This was the only breaking change of updating to [RxJS 7](https://rxjs.dev/deprecations/breaking-changes)